### PR TITLE
new icon: Pulsar (original, original-wordmark)

### LINKED ISSUE #1810: [ICON REQUEST]: Pulsar
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

A popular Community-led Hyper-Hackable Text Editor with GitHub stars: 1.9k

### Links and sources

https://pulsar-edit.dev/

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by weh**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes**
closes #1810 

## Link to prove your SVG is correct and up-to-date.
https://pulsar-edit.dev/

### CODE REVIEW

**Comment by canaleal** [comment]
Hey @weh, the icons and Json looks good, but pulsar is missing from the SVG file names.

**Comment by weh** [comment]
you are right @canaleal, i fixed the filenames :)

**Review by canaleal — COMMENTED** [review]
Everything looks good and follows SVG guidelines. Nice.

File: icons/pulsar/pulsar-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0 1.686-.691c.473-.473.719-1.031.719-1.703V63.813h-2.199v10.324c0 .156-.113.223-.203.223h-6.93a.21.21 0 0 1-.223-.223V63.813h-2.176zm52.957.018v12.699h2.176v-4.434h3.363l.023.047 3.699 4.387h2.445v-.469l-3.32-3.941h.896a2.46 2.46 0 0 0 1.729-.736 2.3 2.3 0 0 0 .719-1.68V66.23c0-.648-.246-1.23-.719-1.68a2.37 2.37 0 0 0-1.703-.721h-9.309zm-94.699.428a5.21 5.21 0 0 0-5.207 5.199 5.21 5.21 0 0 0 5.207 5.195c2.871 0 5.203-2.332 5.203-5.195a5.21 5.21 0 0 0-5.203-5.199zm83.328 1.727a.21.21 0 0 1 .021 0h6.91a.21.21 0 0 1 .223.223l-.002 3.809h-7.354v-3.809a.21.21 0 0 1 .201-.223zm13.75.021a.21.21 0 0 1 .02.002h6.91a.21.21 0 0 1 .223.223v3.473a.21.21 0 0 1-.223.223h-6.91a.21.21 0 0 1-.223-.223V66.23a.21.21 0 0 1 .203-.225zm-66.305.002h6.906c.137 0 .227.09.227.223v3.473c0 .133-.09.225-.227.225h-6.906c-.137 0-.227-.092-.227-.225V66.23c0-.133.09-.223.227-.223zm-36.652 7.949c-.359 0-.719.133-.984.402-.406.402-.52 1.008-.293 1.5l-2.781 1.074L0 90.688l.246.246L14.02 79.375l1.098-2.777a1.49 1.49 0 0 0 .539.113 1.34 1.34 0 0 0 .965-.402c.539-.539.539-1.41 0-1.949a1.39 1.39 0 0 0-.988-.402zm-1.254 6.941l-1.371 1.164a33.49 33.49 0 0 0 3.746 2.777l1.953-.852c-1.594-.918-2.871-1.855-4.328-3.09z"/></svg>
```
Comment (Snailedlt) [inline icons/pulsar/pulsar-original-wordmark.svg:1]: `height` and `width` can be removed. the `fill` color can be changed to `#2c3e50`
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#2c3e50" viewBox="0 0 128 128"><path d="M42.797 48L28.98 59.555l-1.078 2.777c-.18-.066-.359-.109-.516-.109a1.34 1.34 0 0 0-.965.402c-.539.535-.539 1.41 0 1.949a1.41 1.41 0 0 0 .986.402c.359 0 .719-.137.984-.402.406-.406.52-1.008.293-1.5l2.781-1.078 11.576-13.75-.246-.246zm-12.795 1.846c-1.431-.036-2.954.122-4.543.482-1.324.293-2.645.719-3.969 1.277-6.416-2.622-12.697-2.329-16.443 1.41-2.895 2.891-3.859 7.348-2.691 12.52.293 1.32.738 2.645 1.277 3.941-2.242 5.512-2.355 10.93.023 14.715l1.188-1.41c-1.613-2.934-1.613-6.898-.199-11.086a32.11 32.11 0 0 0 4.262 6.271l1.143-1.346c-1.906-2.262-3.43-4.68-4.527-7.145 1.434-3.223 3.656-6.473 6.594-9.406 2.871-2.867 6.102-5.105 9.398-6.582 2.422 1.074 4.867 2.574 7.176 4.523l1.348-1.145a33.1 33.1 0 0 0-6.305-4.23c.719-.246 1.414-.449 2.109-.605 3.543-.805 6.641-.516 9.016.785l1.414-1.186c-1.753-1.121-3.884-1.725-6.27-1.785zM13.66 51.584c1.774.025 3.678.375 5.629 1.029-2.961 1.523-5.855 3.629-8.434 6.227-2.625 2.621-4.711 5.508-6.211 8.422-.227-.695-.449-1.391-.605-2.105-1.012-4.594-.227-8.463 2.219-10.904v-.002c1.851-1.848 4.446-2.708 7.402-2.666zm25.686 3.135l-1.17 1.434c.988 1.813 1.367 4.074 1.145 6.605a28.54 28.54 0 0 1 1.393 2.439c.852-4.098.379-7.726-1.367-10.479zm-17.834 3.918a10.85 10.85 0 0 0-10.836 10.818c0 2.082.605 4.032 1.637 5.689l.586-.227a2.91 2.91 0 0 1 .785-1.566c.09-.09.18-.156.289-.246a8.31 8.31 0 0 1-.852-3.648c0-4.617 3.77-8.379 8.391-8.379a8.32 8.32 0 0 1 3.656.852c.066-.09.133-.199.223-.289a2.77 2.77 0 0 1 1.57-.785l.246-.582c-1.66-1.031-3.609-1.637-5.695-1.637zm12.646 2.334l-1.162 1.342c1.926 2.305 3.453 4.746 4.527 7.145l-.02.066c-1.48 3.293-3.748 6.516-6.553 9.316-4.129 4.121-8.996 6.965-13.727 8.039-3.543.805-6.641.516-9.016-.785l-1.414 1.188c1.859 1.188 4.125 1.793 6.66 1.793 1.324 0 2.715-.156 4.152-.492 5.043-1.145 10.227-4.143 14.578-8.486 2.555-2.555 4.688-5.441 6.234-8.422 1.75 5.195 1.322 10.055-1.639 13.012-2.242 2.242-5.766 3.07-9.824 2.418a25.38 25.38 0 0 1-2.309 1.277c1.727.449 3.387.693 4.934.693 3.41 0 6.35-1.072 8.436-3.154 5.426-5.422 3.609-16.195-3.859-24.949zm-3.428 2.795l-.586.227a2.91 2.91 0 0 1-.785 1.566c-.09.09-.18.156-.289.246a8.32 8.32 0 0 1 .852 3.652c-.023 4.633-3.793 8.375-8.41 8.375a8.24 8.24 0 0 1-3.656-.852c-.07.09-.137.203-.227.289a2.77 2.77 0 0 1-1.57.785l-.223.582c1.66 1.031 3.609 1.637 5.695 1.637 5.969 0 10.836-4.859 10.836-10.816a10.74 10.74 0 0 0-1.637-5.691zm46.43.023v12.742h10.363v-2.195h-8.168V63.789H77.16zm13.793 0a2.38 2.38 0 0 0-1.703.695 2.28 2.28 0 0 0-.695 1.699v2.668a2.28 2.28 0 0 0 .695 1.699c.469.473 1.031.693 1.703.693h6.93c.09 0 .203.068.203.229v2.664c0 .156-.113.223-.203.223h-6.93a.21.21 0 0 1-.223-.223v-.898h-2.176v.898c0 .672.221 1.23.693 1.703a2.3 2.3 0 0 0 1.703.689h6.932a2.35 2.35 0 0 0 1.684-.689c.473-.473.719-1.031.719-1.703v-2.664a2.31 2.31 0 0 0-.719-1.703 2.28 2.28 0 0 0-1.684-.695h-6.932a.21.21 0 0 1-.223-.223v-2.668a.21.21 0 0 1 .223-.223h6.932c.09 0 .203.066.203.223v.896h2.197v-.896a2.3 2.3 0 0 0-.719-1.699c-.449-.449-1.029-.695-1.682-.695h-6.93zm13.906 0a2.38 2.38 0 0 0-1.703.695 2.28 2.28 0 0 0-.693 1.699v10.328h2.176v-4.35h7.354v4.35h2.199V66.184a2.3 2.3 0 0 0-.719-1.699 2.43 2.43 0 0 0-1.703-.695h-6.91zm-54.996.023v12.699h2.176v-4.414h.156l6.977-.023c.672 0 1.23-.246 1.703-.715s.719-1.031.719-1.68v-3.473c0-.648-.246-1.23-.719-1.68-.449-.469-1.031-.715-1.684-.715h-9.328zm13.391 0v10.324c0 .672.221 1.23.693 1.703a2.3 2.3 0 0 0 1.703.691h6.93a2.35 2.35 0 0 0 1.686-.691c.473-.473.719-1.031.719-1.703V63.813h-2.199v10.324c0 .156-.113.223-.203.223h-6.93a.21.21 0 0 1-.223-.223V63.813h-2.176zm52.957.018v12.699h2.176v-4.434h3.363l.023.047 3.699 4.387h2.445v-.469l-3.32-3.941h.896a2.46 2.46 0 0 0 1.729-.736 2.3 2.3 0 0 0 .719-1.68V66.23c0-.648-.246-1.23-.719-1.68a2.37 2.37 0 0 0-1.703-.721h-9.309zm-94.699.428a5.21 5.21 0 0 0-5.207 5.199 5.21 5.21 0 0 0 5.207 5.195c2.871 0 5.203-2.332 5.203-5.195a5.21 5.21 0 0 0-5.203-5.199zm83.328 1.727a.21.21 0 0 1 .021 0h6.91a.21.21 0 0 1 .223.223l-.002 3.809h-7.354v-3.809a.21.21 0 0 1 .201-.223zm13.75.021a.21.21 0 0 1 .02.002h6.91a.21.21 0 0 1 .223.223v3.473a.21.21 0 0 1-.223.223h-6.91a.21.21 0 0 1-.223-.223V66.23a.21.21 0 0 1 .203-.225zm-66.305.002h6.906c.137 0 .227.09.227.223v3.473c0 .133-.09.225-.227.225h-6.906c-.137 0-.227-.092-.227-.225V66.23c0-.133.09-.223.227-.223zm-36.652 7.949c-.359 0-.719.133-.984.402-.406.402-.52 1.008-.293 1.5l-2.781 1.074L0 90.688l.246.246L14.02 79.375l1.098-2.777a1.49 1.49 0 0 0 .539.113 1.34 1.34 0 0 0 .965-.402c.539-.539.539-1.41 0-1.949a1.39 1.39 0 0 0-.988-.402zm-1.254 6.941l-1.371 1.164a33.49 33.49 0 0 0 3.746 2.777l1.953-.852c-1.594-.918-2.871-1.855-4.328-3.09z"/></svg>
```

File: icons/pulsar/pulsar-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
... 43.336-25.266 7.598-7.602 13.934-16.203 18.531-25.07 5.199 15.469 3.934 29.934-4.867 38.734-6.664 6.668-17.133 9.133-29.199 7.199-2.266 1.402-4.531 2.668-6.867 3.801 5.133 1.336 10.066 2.066 14.668 2.066 10.133 0 18.867-3.199 25.066-9.398 16.133-16.133 10.734-48.199-11.465-74.266zm-10.203 8.332l-1.73.668c-.27 1.73-1.066 3.332-2.336 4.664-.266.27-.531.467-.867.732 1.602 3.266 2.535 7 2.535 10.867-.066 13.801-11.266 24.936-25 24.936-3.934 0-7.602-.934-10.867-2.535-.199.27-.398.602-.668.867-1.266 1.266-2.863 2.066-4.664 2.332l-.668 1.734c4.933 3.067 10.735 4.867 16.934 4.867 17.734 0 32.199-14.465 32.199-32.199 0-6.199-1.801-12-4.867-16.934zm-27.398 1.465c-8.535 0-15.469 6.934-15.469 15.469a15.48 15.48 0 0 0 15.469 15.465 15.48 15.48 0 0 0 15.465-15.465c0-8.535-6.934-15.469-15.465-15.469zM46.465 77.266c-1.071 0-2.133.402-2.93 1.199-1.203 1.203-1.535 3-.867 4.469l-8.27 3.199L0 127.066l.734.734 40.934-34.402 3.266-8.266c.531.199 1.066.332 1.602.332 1.066 0 2.066-.398 2.863-1.199 1.602-1.598 1.602-4.199 0-5.801-.797-.797-1.863-1.199-2.934-1.199zm-3.73 20.668l-4.066 3.465c3.598 3.066 7.266 5.801 11.133 8.27 1.398-.535 4.465-1.734 5.801-2.535-4.734-2.734-8.535-5.531-12.867-9.199z"/></svg>
```
Comment (Snailedlt) [inline icons/pulsar/pulsar-original.svg:1]: Same here
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="#2c3e50" viewBox="0 0 128 128"><path d="M127.199 0L86.133 34.398l-3.199 8.27c-.535-.203-1.066-.336-1.535-.336a3.97 3.97 0 0 0-2.863 1.203 4.1 4.1 0 0 0 0 5.797c.797.801 1.865 1.203 2.932 1.203a4.13 4.13 0 0 0 2.934-1.203c1.203-1.199 1.535-3 .867-4.465l8.27-3.199L127.934.734 127.199 0zM87.025 5.486c-3.619.073-7.419.551-11.355 1.447-3.934.867-7.869 2.133-11.803 3.801C44.8 2.933 26.133 3.801 15 14.934 6.398 23.535 3.535 36.801 7 52.199c.867 3.934 2.199 7.867 3.801 11.734-6.668 16.398-7 32.531.066 43.801l3.531-4.199c-4.797-8.734-4.797-20.535-.598-33 3.333 6.464 7.597 12.797 12.664 18.664l3.402-4c-5.668-6.734-10.199-13.934-13.469-21.266 4.27-9.602 10.867-19.266 19.602-28 8.535-8.535 18.133-15.201 27.934-19.604 7.199 3.204 14.464 7.67 21.332 13.471l4-3.402c-6-5.133-12.398-9.332-18.73-12.598 2.133-.734 4.201-1.336 6.268-1.801 10.531-2.398 19.734-1.535 26.801 2.332l4.199-3.531c-5.731-3.668-12.816-5.475-20.777-5.314zM40.602 10.67c5.273.075 10.931 1.115 16.73 3.064-8.797 4.531-17.398 10.801-25.066 18.531-7.801 7.801-14 16.402-18.465 25.066-.668-2.066-1.336-4.133-1.801-6.266-3-13.668-.668-25.199 6.602-32.465 5.498-5.501 13.212-8.057 22-7.932zm76.332 9.332l-3.469 4.264c2.934 5.402 4.07 12.133 3.402 19.668 1.531 2.398 2.934 4.867 4.133 7.266 2.535-12.199 1.133-22.998-4.066-31.197zm-53 11.666c-17.734 0-32.199 14.465-32.199 32.199 0 6.199 1.801 12 4.867 16.934l1.729-.668c.27-1.734 1.068-3.332 2.338-4.668.266-.266.529-.465.865-.73a24.79 24.79 0 0 1-2.535-10.867c0-13.734 11.201-24.934 24.936-24.934a24.62 24.62 0 0 1 10.867 2.531c.199-.266.398-.598.664-.863 1.27-1.27 2.867-2.066 4.668-2.336l.734-1.73c-4.934-3.07-10.734-4.867-16.934-4.867zm37.602 6.934l-3.469 4c5.734 6.863 10.266 14.133 13.469 21.266l-.07.199c-4.399 9.8-11.133 19.398-19.465 27.734-12.266 12.266-26.734 20.734-40.801 23.934-10.531 2.398-19.734 1.531-26.801-2.336l-4.199 3.535c5.535 3.531 12.266 5.332 19.801 5.332 3.934 0 8.066-.465 12.332-1.465 15-3.402 30.402-12.336 43.336-25.266 7.598-7.602 13.934-16.203 18.531-25.07 5.199 15.469 3.934 29.934-4.867 38.734-6.664 6.668-17.133 9.133-29.199 7.199-2.266 1.402-4.531 2.668-6.867 3.801 5.133 1.336 10.066 2.066 14.668 2.066 10.133 0 18.867-3.199 25.066-9.398 16.133-16.133 10.734-48.199-11.465-74.266zm-10.203 8.332l-1.73.668c-.27 1.73-1.066 3.332-2.336 4.664-.266.27-.531.467-.867.732 1.602 3.266 2.535 7 2.535 10.867-.066 13.801-11.266 24.936-25 24.936-3.934 0-7.602-.934-10.867-2.535-.199.27-.398.602-.668.867-1.266 1.266-2.863 2.066-4.664 2.332l-.668 1.734c4.933 3.067 10.735 4.867 16.934 4.867 17.734 0 32.199-14.465 32.199-32.199 0-6.199-1.801-12-4.867-16.934zm-27.398 1.465c-8.535 0-15.469 6.934-15.469 15.469a15.48 15.48 0 0 0 15.469 15.465 15.48 15.48 0 0 0 15.465-15.465c0-8.535-6.934-15.469-15.465-15.469zM46.465 77.266c-1.071 0-2.133.402-2.93 1.199-1.203 1.203-1.535 3-.867 4.469l-8.27 3.199L0 127.066l.734.734 40.934-34.402 3.266-8.266c.531.199 1.066.332 1.602.332 1.066 0 2.066-.398 2.863-1.199 1.602-1.598 1.602-4.199 0-5.801-.797-.797-1.863-1.199-2.934-1.199zm-3.73 20.668l-4.066 3.465c3.598 3.066 7.266 5.801 11.133 8.27 1.398-.535 4.465-1.734 5.801-2.535-4.734-2.734-8.535-5.531-12.867-9.199z"/></svg>
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Excellent work!
Just suggested some simplification of the svg's. Once the suggestions are accepted it's ready to merge!

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️ 
Thank you for another great contribution! 💯 🙇

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Whoops, went through your PR a little fast.
It looks like there are some new conflicts in `devicon.json` could you try and fix them?

**Comment by weh** [comment]
Rebase is done, should be ok now

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**